### PR TITLE
terraform: Update indents

### LINKED ISSUE #15988: Terraform & HCL Treesitter SCM for Indents
### Check for existing issues

- [X] Completed

### Describe the bug / provide steps to reproduce it

When using the Terraform extension, the improper treesitter SCM is adding an unnecessary indent before the closing bracket `}`.

The result is this:

```hcl
terraform {

  }
```

When it should be this:

```hcl
terraform {

}
```

The SCM is located `extensions/terraform/languages/terraform/indents.scm`

The comment points to [this nvim-treesitter reference](https://github.com/nvim-treesitter/nvim-treesitter/blob/ce4adf11cfe36fc5b0e5bcdce0c7c6e8fbc9798a/queries/hcl/indents.scm)

But the content doesn't match. I assume this is slight oversight. When you manually replace the indents.scm, the issue is resolved.

### Environment

N/A

### If applicable, add mockups / screenshots to help explain present your vision of the feature

https://github.com/user-attachments/assets/3340ef36-57d9-4521-bbf3-2432802e0587

https://github.com/user-attachments/assets/bd58dd7b-cd37-4c43-822b-5916bb35ba75

### If applicable, attach your Zed.log file to this issue.

<details><summary>Zed.log</summary><pre>

</pre></details>

### PR DESCRIPTION
**Pull request by taras-mrtn**
Closes #15988

Fixed indent configuration for terraform/HCL

Release Notes:
- N/A

https://github.com/user-attachments/assets/09b44ac9-ef09-463c-876d-0fbcdd1f09c9

### CODE REVIEW

File: extensions/terraform/extension.toml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 id = "terraform"
 name = "Terraform"
 description = "Terraform support."
-version = "0.0.4"
+version = "0.0.5"
```
Comment (maxdeviant) [inline extensions/terraform/extension.toml:4]: We bump the extension versions separately from changes:

```suggestion
version = "0.0.4"
```

**Comment by taras-mrtn** [comment]
@mikayla-maki can you please take a look at this PR?

**Comment by mikayla-maki** [comment]
Thanks!

**Comment by taras-mrtn** [comment]
@mikayla-maki for some reason I don't see this fix in either the 0.153 or 0.154 preview. Can you help with this?

**Comment by taras-mrtn** [comment]
Added PR to bump version https://github.com/zed-industries/zed/pull/18382

**Pull request merged**